index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="menu-container">
+            <a href="#" class="logo">My Logo</a>
+            <nav class="navbar">
+                <img src="./images/Union.png" alt="menu">
+            </nav>
+    </header>
+    <section class="main">
+        <h1 class="heading1">I'm Manzi. <br/>Glad to see <br/>you!</h1>
+        <p class="bio"> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <h3 class="conn">LET’S CONNECT</h3>
+        <div class="media-container">
+            <a href="#"><img src="./images//Vector.png"></a>
```

Comment:
- [ ] please put back the media icons in a ul tag to group them, and li per icon as it was . You can display them flex and give them a direction of row.
Using li and ul is not only more semantic, but it gives meaning to the html nodes, helping screen readers, web crawlers, etc